pyvelox/pyvelox.h

```diff
@@ -416,6 +417,25 @@ inline void addVectorBindings(
   m.def("from_list", [](const py::list& list) mutable {
     return pyListToVector(list, PyVeloxContext::getInstance().pool());
   });
+
+  m.def("export_to_arrow", [](VectorPtr& inputVector) {
+    auto arrowArray = new ArrowArray();
+    std::shared_ptr<facebook::velox::memory::MemoryPool> pool_{
+        facebook::velox::memory::getDefaultMemoryPool()};
+    facebook::velox::exportToArrow(inputVector, *arrowArray, pool_.get());
+    inputVector.reset();
```

Comment:
It was with reference to the usage comment here,
https://github.com/facebookincubator/velox/blob/main/velox/vector/arrow/Bridge.h#L51

But, now I think it will be better to not reset it here, since it maybe used later by the user, even after conversion.

Removed it in the new commits.